src/System.Web.WebPages/Helpers/AntiXsrf/AntiForgeryWorker.cs

```diff
@@ -104,7 +104,11 @@ public TagBuilder GetFormInputElement(HttpContextBase httpContext)
                 // Adding X-Frame-Options header to prevent ClickJacking. See
                 // http://tools.ietf.org/html/draft-ietf-websec-x-frame-options-10
                 // for more information.
-                httpContext.Response.AddHeader("X-Frame-Options", "SAMEORIGIN");
+                var xFrameHeaderName = "X-Frame-Options";
```

Comment:
Oh actually one tiny comment: this should be a `const string` because it's not really a variable.

Reply:
Please also rename this `const` because the command-line build hits an FxCop error:
```
SA130: The variable name 'xFrameHeaderName' begins with a prefix that looks like Hungarian notation. Remove the prefix or add it to the list of allowed prefixes.
```